docs/guides/security/authentication/active_directory_authentication.md

```diff
@@ -134,6 +122,19 @@ for an Active Directory user.
 administrator@ad.company.local:*:1450400500:1450400513:Administrator:/home/administrator@ad.company.local:/bin/bash
 ```
 
+!!! Note 
+
+    `getent` get entries from Name Service Switch libraries (NSS), it means that, contrary to `passwd` or `dig` for example, it will ask for responses in different databases, including `/etc/hosts` for `getent hosts` or from `sssd` in the `getent passwd` case.
```

Comment:
1. to smooth this, either make this two sentences or remove the word "it":

'getent' is the command to get entries from Name Service Switch libraries (NSS). It means that...
OR
'getent', or get entries from Name Service Switch libraries (NSS), means that...

2. does it mean the same thing and is it a familiar enough term if you smooth the end of this sentence by saying, "it will query different databases" rather than "it will ask for responses in different databases"?